todo_app/data/trello_items.py

```diff
@@ -64,16 +76,19 @@ def change_item_status(id, new_status):
         new_status: the status to be changed too
 
     """
+
+    auth_payload = get_auth_payload(config)
+
     list_id = get_list_id_with_list_name(new_status)
```

Comment:
Here you changed this method to expect a config argument but don't pass it through, so your behaviour to change lists errors